subprocess/subtree_oom.go

```diff
@@ -63,3 +68,58 @@ func getPidFromLog(line string) (int, bool) {
 	}
 	return pid, true
 }
+
+func ClearOOMHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		grip.Debug("clearing oom")
+		resp := NewOOMTracker()
+		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
```

Comment:
there's a context in request.Context() that you should use rather than background.